lib/widgets/autocomplete.dart

```diff
@@ -212,8 +250,16 @@ class ComposeAutocomplete extends AutocompleteField<MentionAutocompleteQuery, Me
           children: [
             avatar,
             const SizedBox(width: 8),
-            Text(label),
-          ])));
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+              Text(label, style: TextStyle(fontSize: 18,  height: 1.1, color: designVariables.contextMenuItemLabel) //Creates a line height equavalent to ~20
+                  .merge(weightVariableTextStyle(context, wght: 600))),
+              Visibility(
+                visible: metaData.isNotEmpty,
+                child: Text(metaData, style: TextStyle(height: 1.14, color: designVariables.contextMenuItemMeta)), //Creates a line height equavalent to ~16
+              )])])));
```

Comment:
indentation and line lengths:

```suggestion
            Column(
              mainAxisSize: MainAxisSize.min,
              crossAxisAlignment: CrossAxisAlignment.start,
              children: [
                Text(
                  style: TextStyle(
                    fontSize: 18,
                    height: 1.1, // Creates a line height equavalent to ~20
                    color: designVariables.contextMenuItemLabel,
                  )
                    .merge(weightVariableTextStyle(context, wght: 600)),
                  label),
                Visibility(
                  visible: metaData.isNotEmpty,
                  child: Text(
                    style: TextStyle(
                      height: 1.14, // Creates a line height equavalent to ~16
                      color: designVariables.contextMenuItemMeta,
                    ),
                    metaData)),
              ]),
          ])));
```